version/version.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Why do we need trhis file? Why isn't the version in GIE good enough?

The `go build` in the Dockerfile actually updates the version fields in GIE.